cmd/meta/main.go

```diff
@@ -188,16 +193,16 @@ func (m *Main) RunDump(args Args) error {
 }
 
 func (m *Main) RunServer(args Args) error {
-	var musicbrainzDB *sql.DB
-	if uri := args.String("--musicbrainz-index"); uri != "" {
+	var DB *sql.DB
+	if uri := args.String("--index"); uri != "" {
```

Comment:
The MusicBrainz and CWR index DB should be separate files.

Reply:
Could you please clarify more?
They are separated.The server support both MusicBrainz and CWR APIs .

Reply:
The `--musicbrainz-index` flag is intended to point at a SQLite3 database file containing just the MusicBrainz index. You have removed that in favour of `--index` which you are expecting to be both the MusicBrainz and the CWR database, but they should be separate files so require separate flags.